pkg/types/defaults/installconfig.go

```diff
@@ -129,4 +134,32 @@ func SetInstallConfigDefaults(c *types.InstallConfig) {
 	if c.AdditionalTrustBundlePolicy == "" {
 		c.AdditionalTrustBundlePolicy = types.PolicyProxyOnly
 	}
+
+	setPullSecretDefault(c)
+}
+
+func setPullSecretDefault(config *types.InstallConfig) {
+	if config.PullSecret != "" {
+		return
+	}
+
+	var pullSecretFile string
+	if pullSecretFile = os.Getenv("REGISTRY_AUTH_FILE"); pullSecretFile == "" {
+		return
+	}
+
+	var err error
+	if _, err = os.Stat(pullSecretFile); err != nil {
+		logrus.Warn("REGISTRY_AUTH_FILE is set, but the credentials file does not exist")
```

Comment:
Can we include the error here, to help debugging?  I'd expect  the error to include the path being attempted, and also a description of why the attempt failed, although it's probably worth checking to confirm

Reply:
and actually, why is this log-and-continue instead of returning a (wrapped?) error?